components/user/UserHorizontal.js

```diff
@@ -42,7 +43,7 @@ export default function UserHorizontal({ profile }) {
               if (!trimmedTag) {
                 return null;
               }
-              return <TagSimple name={trimmedTag} key={index} />;
+              return <TagSimple name={trimmedTag} key={index} isSelected={searchTagNameInInput(input, trimmedTag)} />;
```

Comment:
Formatting isn't correct here, this will cause issues in future pull requests

```suggestion
              return (
                <TagSimple
                  name={trimmedTag}
                  key={index}
                  isSelected={searchTagNameInInput(input, trimmedTag)}
                />
              );
```